domehunter/dome_control.py

```diff
@@ -179,15 +180,15 @@ def __init__(self,
 
         # TODO: read in default value from yaml(?)
         self._degrees_per_tick = Angle(degrees_per_tick * u.deg)
-        self.az_position_tolerance = Angle(az_position_tolerance * u.deg)
-        # if the requested tolerance is less than degrees_per_tick use
-        # degrees_per_tick for the tolerance
-        self.az_position_tolerance = max(
-            self.az_position_tolerance, self.degrees_per_tick)
+        self._az_position_tolerance = Angle(az_position_tolerance * u.deg)
```

Comment:
As raise in the error instead of just using the degrees_per_tick and sweeping it under the rug? Or should we still change the tolerance to be the same as degrees_per_tick but log some kind of warning message?